pyanaconda/dbus/observer.py

```diff
@@ -228,43 +228,3 @@ def __repr__(self):
         """Returns a string representation."""
         return "{}({})".format(self.__class__.__name__,
                                self._service_name)
-
-
-class PropertiesCache(object):
```

Comment:
It would be great to specify why "it doesn't look" that the properties cache is not required anymore. Commit 943d9f00a1bb83fe9f594f49657de0d48f7f7807 .

Reply:
Fixed.